Code/Framework/AzToolsFramework/AzToolsFramework/ComponentModes/CapsuleViewportEdit.cpp

```diff
@@ -0,0 +1,205 @@
+/*
```

Comment:
question: Is it possible to run clang-format on this file too? Would be nice if possible (maybe you have already? 🙂)

Reply:
I was running it on sections as I went. I just ran it again on the whole file but I didn't notice a lot of change.

Reply:
Okay cool sounds good! 🙂